specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/nova.json

```diff
@@ -0,0 +1,1051 @@
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/clusterNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted. Deletion will run asynchronously."
+          },
+          "200": {
+            "description": "Succeeded. The managed Cassandra cluster has been deleted."
+          },
+          "204": {
+            "description": "The specified cluster does not exist."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/DefaultErrorResponse"
+            }
+          }
+        }
+      },
+      "put": {
+        "operationId": "CassandraCluster_CreateUpdate",
+        "description": "Creates a managed Cassandra cluster.",
+        "x-ms-examples": {
+          "CosmosDBManagedCassandraClusterCreateHybrid": {
+            "$ref": "./examples/CosmosDBManagedCassandraClusterCreateHybrid.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/clusterNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ClusterResource"
+            },
+            "description": "Parameters to provide for specifying the managed Cassandra cluster."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The managed Cassandra cluster is being updated. Poll for provisioningState=Succeeded to find when creation completes.",
+            "schema": {
+              "$ref": "#/definitions/ClusterResource"
+            }
+          },
+          "201": {
+            "description": "The managed Cassandra cluster is being created. Poll for provisioningState=Succeeded to find when creation completes.",
+            "schema": {
+              "$ref": "#/definitions/ClusterResource"
+            }
+          },
```

Comment:
202 -> request is accepted and will complete async
200 -> OK, no-op (I think for our scenarios 200 and 202 can be used interchangeably) Although, I think 202 is more correct. 
201-> created 
Can you please make the descriptions a little more clear.

Reply:
We can't return 200 when the request is completed, because a put is a request. We can use it for when the request is a no-op, and I've added that.